introduction.md

```diff
@@ -0,0 +1 @@
+I am a developer who uses Git everyday. It is nice to know more about it. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.